app/assets/stylesheets/utilities/layout.scss

```diff
@@ -9,3 +9,16 @@
 .justify-content-between {
   justify-content: space-between;
 }
+
+.flash-holder {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.flash-message {
+  width: 100%;
+  padding: 1em;
+  color: #ffffff;
```

Comment:
Do we have a css linter?
If not, we should introduce one soon.

Think it fail on this as it can be just "#fff"

Reply:
I'm hoping we don't have to write much custom CSS - I'm not great at it, a linter would probably help!

Reply:
Looks like we can do it as part of govuk-lint:
https://github.com/alphagov/govuk-lint#sass